### DISCUSSION TITLE
github student's developer pack

### DISCUSSION THREAD

**Discussion by Divycholera98** [community/community#49704] [GitHub Education]
for the verification purpose , i Have a id card having valid date on the back side and the college name on the front and due to which it is not able to verify my document. and there is no option of multiple uploads so what can i do next?

**Comment by samus-aran** [comment]
Hi @Divycholera98 👋 

We're happy you're here! We do understand that some of our students don't have dated Student ID but we do prefer Student IDs so please continue to use Student ID as evidence. We would also recommend you take a photo of the Student ID along side another piece of evidence if necessary.

Let us know how you get on 👍
